build/media-suite_compile.sh

```diff
@@ -1228,7 +1228,7 @@ if [[ $x264 != no ]]; then
         unset_extra_script
         if [[ $standalone = y && $x264 =~ (full|fullv) ]]; then
             _check=("$LOCALDESTDIR"/opt/lightffmpeg/lib/pkgconfig/libav{codec,format}.pc)
-            do_vcs "https://git.ffmpeg.org/ffmpeg.git"
+            do_vcs "https://github.com/FFmpeg/FFmpeg.git"
```

Comment:
I'm pretty sure this mirror is not automatically synced. As far as the suite is concerned that might be good enough, but for people that need to be directly using the official repo it might be better to leave this as it was and maybe prioritize repo fallbacks feature for this one.

Reply:
Sure, I can change it back to ffmpeg.org or change it to the gitlab since that is updated hourly

Reply:
If you feel like not doing the fallbacks soonish, I'm also personally fine with still changing this URL to the github mirror as I'm not building the suite as frequently as I used to (my main driver now is a Mac as well).

Reply:
I'll probably start of the fallback function after I get home so I can do that then. TBH, I used to have only windows computers (laptop, 3 desktops, and 2 server 2012 r2,) but now I only have one windows desktop with everything else Linux based